docs/gaming/distribution.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Distribution
+head:
+  - - meta
+    - name: title
+      content: Solana Cookbook | Game Distribution
+  - - meta
+    - name: og:title
+      content: Solana Cookbook | Game Distribution
+  - - meta
+    - name: description
+      content: Different options on how to distribute your game built on Solana
+  - - meta
+    - name: og:description
+      content: Different options on how to distribute your game built on Solana
+  - - meta
+    - name: og:image
+      content: https://solanacookbook.com/cookbook-sharing-card.png
+  - - meta
+    - name: og:image:alt
+      content: Solana splash card
+  - - meta
+    - name: twitter:card
+      content: summary
+  - - meta
+    - name: twitter:site
+      content: "@solanacookbook"
+  - - meta
+    - name: twitter:image
+      content: "https://solanacookbook.com/cookbook-sharing-card.png"
+  - - meta
+    - name: robots
+      content: index,follow,noodp
+  - - meta
+    - name: googlebot
+      content: index,follow
+footer: MIT Licensed
+---
+
+# Distribution
+
+Distribution of your game depends highly on the platform you are using of course. With Solana game SDKs you can build for IOS, Android, Web and Native Windows or Mac. Using the Unity SDK you could even connect Nintendo Switch or XBox to Solana theoretically. Many game companies are pivoting to a mobile first approach because there are so many people with mobile phones in the world. Mobile comes with its own complications though, so you should pick what fits best to your game.  
+
+A unique advantage that Solana has compared to other blockchains, is that Solana is selling a crypto native mobile phone called [Saga](https://solanamobile.com/) which comes with a new innovative dApps store which allows you to distribute crypto games without the restrictions on traditional app stores from for examples Google or Apple. 
+
+## Publishing Platforms
+
+Platforms where you can host your games
+
+| Platform | Description | Link |
+| --- | --- | --- |
+| Fractal | A game publishing platform that supports Solana and Ethereum. They have an amazing team and are super supportive on bringing on new good games. They also have their own wallet and account handling and there is an SDK for high scores and tournaments  | https://www.fractal.is/ |
+| Elixir | Platform for web3 games that also offers a PC launcher | https://elixir.app/ |
+| Vercel | Just host your game yourself. Can be easily setup so that a new version get deployed as soon as you push to your repository.  | https://vercel.com/ |
+| Solana mobile DApp Store | The Solana alternative to the Play and AppStore. A crypto first variant of a dApp store, which is open source free for everyone to use. Applications for the release of the dApp store a open. | https://github.com/solana-mobile/dapp-publishing/blob/main/README.md |
```

Comment:
`Applications for the release of the dApp store a open` - what does this mean?

Reply:
`alternative to the Play and AppStore` -> `alternative to Google Play and the Apple App Store`